Info-Orbs/src/widgets/clockWidget.cpp

```diff
@@ -49,11 +50,12 @@ void ClockWidget::draw(bool force) {
 }
 
 void ClockWidget::displayAmPm(uint32_t color) {
+    GlobalTime* time = GlobalTime::getInstance();
     m_manager.selectScreen(2);
     TFT_eSPI& display = m_manager.getDisplay();
     display.setTextSize(4);
     display.setTextColor(color, TFT_BLACK, true);
-    const String& am_pm = m_hourSingle >= 12 ? "AM" : "PM";
+    const String& am_pm = time->isPM() ? "PM" : "AM";
```

Comment:
why `const String &` instead of just `String`

Reply:
It was just to match the types of `display.drawString`